legal-api/migrations/versions/4aed1fbbba29_.py

```diff
@@ -24,15 +24,15 @@ def upgrade():
                     sa.Column('foreign_jurisdiction_region', sa.String(length=10), nullable=True),
                     sa.Column('expiry_date', sa.DateTime(timezone=True), nullable=True),
                     sa.Column('legal_entity_id', sa.Integer(), nullable=True),
-                    sa.Column('change_filing_id', sa.Integer(), nullable=True),
-                    sa.ForeignKeyConstraint(['change_filing_id'], ['filings.id'], ),
+                    sa.Column('filing_id', sa.Integer(), nullable=True),
```

Comment:
@argush3 did we already updated db with these versions? do we need to update db manually for now? These migration is obsolete once @thorwolpert replace this with business-model